test/Microsoft.IdentityModel.JsonWebTokens.Tests/JsonWebTokenHandler.ValidateTokenAsyncTests.Audience.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.TestUtils;
+using Microsoft.IdentityModel.Tokens;
+using Xunit;
+
+namespace Microsoft.IdentityModel.JsonWebTokens.Tests
+{
+    public partial class JsonWebTokenHandlerValidateTokenAsyncTests
```

Comment:
It's not clear in the VS test runner which test this is for, not sure how this will look w/the one for lifetime
<img width="276" alt="image" src="https://github.com/user-attachments/assets/f8291dbf-6506-4a13-a1b1-a0505d98f814">

Reply:
You are looking at the previous regression tests, which I am decoupling into separate files.
The current one shows as:

<img width="378" alt="Screenshot 2024-09-24 at 12 29 28" src="https://github.com/user-attachments/assets/cc7f7457-385f-4215-b31b-c9ab97279e05">

As part of the next PRs, I am removing `JsonWebTokenHandlerValidationParametersTests`, and using partial classes to keep all the different aspects inside the same class in the text explorer.
We are looking at having `ValidateTokenAsync_Audience`, `ValidateTokenAsync_Lifetime`, and so on, all under `JsonWebTokenHandlerValidateTokenAsyncTests`.

Reply:
If the goal is to find the Audience tests, then a different ns such as 'Microsoft.IdentityModel.JsonWebTokens.Audience.Tests' would create a hierarchy that would have one drop down for each validation step.

Reply:
It moves them under the same type of class. I can see why this would be helpful, but it's also not the style currently used in the library, so that could cause some confusion. Might be easier to just update the namespace and keep the consistency. Sometimes, having to context switch, even for something small like that, can add up.